workbench-core/accounts/src/models/accounts/account.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+import { HOSTING_ACCOUNT_STATUS } from '../../constants/hostingAccountStatus';
+import { CostCenterParser } from '../costCenters/costCenter';
+
+// eslint-disable-next-line @rushstack/typedef-var
+const ErrorParser = z.object({
+  type: z.string(),
+  value: z.string()
+});
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const AccountParser = z.object({
+  id: z.string(),
+  name: z.string(),
+  awsAccountId: z.string(),
+  envMgmtRoleArn: z.string(),
+  error: ErrorParser.optional(),
+  hostingAccountHandlerRoleArn: z.string(),
+  vpcId: z.string().optional(),
+  subnetId: z.string().optional(),
+  cidr: z.string().optional(),
```

Comment:
Should this be removed if we plan on managing CIDRs through ETCs? Though I do understand if this is a limiting factor that ETC has to belong within the CIDR defined here.

Reply:
Correct, CIDR can be removed since we don't store CIDR in the account data model. CIDR is stored in ETC and we pull the value from [there](https://github.com/aws-solutions/solution-spark-on-aws/blob/26ee548f9981e1c2d6a2393f11483eba09a7f697/solutions/swb-reference/src/environment/sagemakerNotebook/sagemakerNotebookEnvironmentLifecycleService.ts#L28) when launching a workspace.

Reply:
Updated